ec3730/Views/Interface/InterfaceView.swift

```diff
@@ -0,0 +1,90 @@
+import Foundation
+import NetUtils
+import SwiftUI
+import SystemConfiguration.CaptiveNetwork
+import UIKit
+
+struct InterfaceView: View {
+    @ObservedObject var model: ReachabilityModel
+    let interface: Interface
+
+    var body: some View {
+        List {
+            basicInfo()
+
+            wifiInfo()
+
+            let proxyKeys = [
+                (key: "FTPPassive", title: "FTP Passive"),
+                (key: "ExceptionsList", title: "Exceptions List"),
+                (key: "RTSPEnable", title: "RTSP"),
+                (key: "GopherEnable", title: "Gopher"),
+                (key: "HTTPEnable", title: "HTTP"),
+                (key: "FTPEnable", title: "FTP"),
+                (key: "HTTPSEnable", title: "HTTPS"),
+                (key: "HTTPSProxy", title: "HTTPS Proxy"),
+                (key: "ProxyAutoConfigEnable", title: "Proxy Auto Configuration"),
+                (key: "HTTPSPort", title: "HTTPS Port"),
+                (key: "SOCKSEnable", title: "SOCKS"),
+                (key: "ProxyAutoDiscoveryEnable", title: "Proxy Auto Discovery"),
+                (key: "HTTPProxy", title: "HTTP Proxy"),
+                (key: "ExcludeSimpleHostnames", title: "Excludes Simple Hostnames"),
+            ]
+
+            if let proxyInfo = proxyInformation {
+                ForEach(proxyKeys, id: \.key) { item in
+                    if let value = proxyInfo[item.key] as? String {
```

Comment:
I think it will be good to make it a function that returns the cell for item.

Reply:
Complete in [a9dd7f3](https://github.com/twodayslate/NetUtils/pull/99/commits/a9dd7f32f1d8d7830ab932a4c6372d4453ade3d3)